key-vault.tf

```diff
@@ -1,18 +1,18 @@
 module "vault" {
-  source = "git@github.com:hmcts/cnp-module-key-vault?ref=master"
-  name = "${var.product}-${var.env}"
-  product = "${var.product}"
-  env = "${var.env}"
-  tenant_id = "${var.tenant_id}"
-  object_id = "${var.jenkins_AAD_objectId}"
-  resource_group_name = "${azurerm_resource_group.rg.name}"
-  product_group_object_id = "${var.product_group_object_id}"
-  managed_identity_object_id = "${var.managed_identity_object_id}"
-  common_tags = "${local.common_tags}"
+  source                      = "git@github.com:hmcts/cnp-module-key-vault?ref=azurermv2"
+  name                        = join("-", [var.product, var.env])
+  product                     = var.product
+  env                         = var.env
+  tenant_id                   = var.tenant_id
+  object_id                   = var.jenkins_AAD_objectId
+  resource_group_name         = azurerm_resource_group.rg.name
+  product_group_object_id     = var.product_group_object_id
+  managed_identity_object_id  = var.managed_identity_object_id
+  common_tags                 = local.common_tags
 }
```

Comment:
@vijayrajagopalan-hmcts  dont u need to set create_managed_identity = true??

Reply:
Yes @shashank-rastogi. The new MI will be created by the Key Vault module.